pages/docs/index.js

```diff
@@ -62,6 +62,16 @@ export default function DocsIndex() {
             color: "bg-orange-100 text-orange-800",
           },
         },
+        {
+          name: "LinkFree CLI",
+          path: "/docs/environments/linkfree-cli",
+          description:
+            "This CLI tool will allow you to create profile, update profile, add testimonal and more from the command line.",
```

Comment:
Just minor grammar improvements. 🙂
```suggestion
            "This CLI tool will allow you to create a profile, update your profile, add testimonials, and more from the command line.",
```

Reply:
The current sounds good. If there is something, it can come in another PR.

Thanks for the review